### PR TITLE
Added U2F rules

### PR DESCRIPTION
**Pull request by romanstingler**
Just to be productive and wait until someone fixes #122

### CODE REVIEW

**Comment by keszybz** [comment]
That files is from https://github.com/Yubico/libu2f-host/blob/master/70-u2f.rules. It's totally OK to keep udev rules decentralized. Systemd developers don't know too much about yubico device numbers, and if you want to use the hardware, you are likely to install their package anyway. So if this was to be moved to systemd, some good justification would be required.

(The Fedora package does not install the udev rules file. I don't know why…)

The other two commits need to be squashed into one?

**Comment by keszybz** [comment]
Ah, OK. So Fedora packages https://github.com/amluto/u2f-hidraw-policy/, which is described as
> Unlike the Yubico sample udev rule, this should work for any compliant U2F device, including non-Yubico devices and as-yet-unreleased devices.

So it seems there's no need for the yubico udev rules.

**Comment by poettering** [comment]
hmm, yeah, the U2F part looks misplaced, that should remain in some other non-systemd package that deals with U2F devices.

The gaming mouse stuff, is that something that should go in? /cc @whot

**Comment by keszybz** [comment]
> U2F part [...] should remain in some other non-systemd package that deals with U2F devices.

To clarify: yubico upstream has rules which require updating for each device number, so if you plug in a very new device, it won't be supported. `u2f-hidraw-policy` has a helper and rules, which, UUIC, provide generic support. So the rules in this PR are a) managed by yubico and should stay in their repo, b) semi-obsolete because a better solution exists.

File: hwdb/70-mouse.hwdb [hunk 1]
```diff
@@ -507,4 +507,4 @@ mouse:usb:v145fp01ac:name:HID-compliant Mouse Trust Gaming Mouse:
 
 # Zelotes 5500 DPI 7 Button USB Wired Gaming Mouse
 mouse:usb:v1d57pad17:*
- MOUSE_DPI=*1000@500 1600@500 2400@500 3200@500 5500@500
+ MOUSE_DPI=1000@500 1600@500 2400@500 3200@500 5500@500 *1000@1000 1600@1000 2400@1000 3200@1000 5500@1000
```
Comment (whot) [inline hwdb/70-mouse.hwdb]: is the report rate software-controlled or is it a physical hardware switch? If the latter, we can add it. In the former case we don't add them unless the 1000Hz is the out-of-the-box setting (in which case we'd have to remove the 500Hz)

**Comment by poettering** [comment]
I figure we can close this issue. The PR was about U2F, but that doesnt look right. The gaming mouse stuff appears unrelated to that, and if it still relevant, should be resubmitted as a new, fresh, separate PR.

Closing hence.

@romanstingler please resubmit the gaming mouse bits as a new PR, if this is still relevant. Thanks

**Pull request closed without merging**